src/utils/__tests__/style-formatter.spec.js

```diff
@@ -1,24 +1,21 @@
 import styleFormatter from '../style-formatter';
-import defaultValues from '../../__tests__/default-button-props';
+import defaultProperties from '../../__tests__/default-button-props';
 
 describe('style-formatter', () => {
   describe('getStyles', () => {
     let style;
     const defaultStyle =
-      'width: 100%;height: 100%;padding: 4px;color: #ffffff;font-size: 12px;text-align: center;background-color: #4477aa;cursor: pointer;border: none;';
+      'width: 100%;height: 100%;padding: 4px;cursor: pointer;font-size: 12px;color: #ffffff;font-weight: bold;text-align: center;background-color: #4477aa;border: none;';
     const someColor = '#ffff00';
     const someColorExpression = 'rgb(255,255,0)';
     const someSize = 24;
     const someUrl = '/media/Logo/qlik.png';
-    const { Theme } = defaultValues;
+    const { Theme, layout } = defaultProperties;
     const disabled = false;
     let element;
 
     beforeEach(() => {
-      style = {
-        border: { isUsed: false },
-        textAlign: 'center',
-      };
+      style = JSON.parse(JSON.stringify(layout.style));
```

Comment:
I checked if this was necessary and it is, just don't understand really why

Reply:
setting an object to a already existing object, makes it a reference, not a copy. So the `beforeEach` does not reset `style`, since `layout.style` also changes